src/reporter.js

```diff
@@ -11,16 +11,33 @@ var ADMIRAL_PROJECT = process.env.ADMIRAL_PROJECT;
 var ADMIRAL_PHASE = process.env.ADMIRAL_PHASE;
 
 // Optional
-var ADMIRAL_RUN = process.env.ADMIRAL_RUN_ID;
+var ADMIRAL_RUN = process.env.ADMIRAL_RUN_ID.replace(%/%g, "-");
```

Comment:
we have concern on this `.replace(%/%g, "-")`. what's the purpose of replacing string here?